contracts/Pod.sol

```diff
@@ -47,76 +47,66 @@ contract Pod is
     |__________________________________*/
     IERC20Upgradeable public token;
     IERC20Upgradeable public ticket;
-    IERC20Upgradeable public pool;
+    IERC20Upgradeable public reward;
 
     // Initialized Contracts
     TokenFaucet public faucet;
     TokenDrop public drop;
 
-    // Private
-    IPrizePool private _prizePool;
-
     // Manager
     IPodManager public manager;
 
+    // Private
+    IPrizePool private _prizePool;
+
     // Factory
     address public factory;
```

Comment:
Let's remove the factory member.  It's only needed because in the PodFactory the `pod.transferOwnership` occurs *before* the `setTokenDrop`.  Just move ownership transfer after and you won't need to set the factory.

Reply:
🧠 Brilliant!